src/ffi/floatobject.rs

```diff
@@ -1,6 +1,8 @@
 use crate::ffi::object::*;
 use std::os::raw::{c_double, c_int};
 
+// TODO: mark non-limited
+// currently used by types/floatob.rs
 #[repr(C)]
 pub struct PyFloatObject {
```

Comment:
Similar issue to `PyDictObject`. This is not a limited API, but is used unconditionally to define `PyFloat` in floatob.rs